cmd/lncli/commands.go

```diff
@@ -1289,14 +1291,15 @@ func parseChannelPoint(ctx *cli.Context) (*lnrpc.ChannelPoint, error) {
 	case args.Present():
 		index, err := strconv.ParseUint(args.First(), 10, 32)
 		if err != nil {
-			return nil, fmt.Errorf("unable to decode output index: %v", err)
+			return nil, nil, fmt.Errorf("unable to decode output index: %v", err)
 		}
 		channelPoint.OutputIndex = uint32(index)
+		args = args.Tail()
```

Comment:
Have all the callers been updated to compensate for this API change? Not immediately clear why this change is required.

Reply:
So our pattern here is to consume an arg then pop it off of args so that we can process what's left. In `parseChannelPoint` we were not popping the output index off of the list of args so code after `parseChanelPoint` would re-process the output index (in my case, try to read it as the start time). 

This wasn't a problem for the other places where it's called (`abandonChannel`/`closeChannel`) because they do not take any further args. They've been updated, but it's not a significant change. 

On second look, this would prob be nicer as `parseChannel(ctx cli.Context, args *cli.Args)` to avoid an extra return item. I was against passing `ctx` and `args` because cli has args but I think that way is better.